### PR TITLE
fix: typescript issue in banner when customizing buttons

### PR DESCRIPTION
**Pull request by ericledonge**
### Summary

This PR fixing this issue: https://github.com/callstack/react-native-paper/issues/1670

With this fix, I can customize button you can pass other props that button component takes, without having TS crying, as described in the doc: https://callstack.github.io/react-native-paper/banner.html#actions

Maybe, the way I've submitted the modification in lib/typescript/components/Banner.d.ts is not good. I just want to change the line 32. I am new to both open source and typescript worlds. So let me know how to submit that correctly.

### Test plan

In my IDE, everything is ok:

Capture d’écran, le 2021-11-07 à 21 30 07

### CODE REVIEW

File: src/components/Banner.tsx [hunk 1]
```diff
@@ -33,10 +33,12 @@ type Props = $RemoveChildren<typeof Surface> & {
...
    * To customize button you can pass other props that button component takes.
    */
-  actions: Array<{
-    label: string;
-    onPress: () => void;
-  }>;
+  actions: Array<
+    {
+      label: string;
+      onPress: () => void;
```
Comment (Drakeoon) [inline src/components/Banner.tsx:39]:
```suggestion
```

I think this line could be removed, since `onPress()` is already covered by `Button` interface

**Review by Drakeoon — CHANGES_REQUESTED** [review]
@ericledonge I think the way you also updated lib files here is fine. @lukewalczak any preferences about it?

I left a quick note, please apply it and I'll give a green light on this one. Imo a really good change 👍🏻

**Comment by Drakeoon** [comment]
Oh, I actually thought we had the lib folder in our repo and it's just an update. I think you should also remove the lib changes, as we compile them for every release on our own @ericledonge

**Comment by ericledonge** [comment]
@Drakeoon Oki, let me know if it's correct now.

**Review by Drakeoon — APPROVED** [review]
@lukewalczak looks good to me 👍🏻  Thanks @ericledonge

**Pull request merged**